Controllers/Groups/GroupsController.cs

```
@@ -34,7 +38,7 @@ public IActionResult GetById(int id)
             var group = _groupRepository.Find(id);
```

I think repository should throw the exception but not controller